quaistats/quaistats.go

```diff
@@ -55,6 +56,10 @@ const (
 
 	// reportInterval is the time interval between two reports.
 	reportInterval = 15
+
+	alpha               = 8
```

Comment:
we should adopt our const standard, please prefix the constant with c_

Reply:
should we do it for the other constants above as well?